index.html

```diff
@@ -11,6 +11,6 @@
     }
   </style>
   <body>
-    <h1 class="some-heading">GIT WORKFLOW WORKSHOW</h1>
+    <h1 class="page-heading">GIT WORKFLOW WORKSHOP</h1>
```

Comment:
Hmm... you also changed the class name. That is out of scope

Reply:
Sorry about that, will be more careful!

Reply:
Alright. You are forgiven.....